specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/DatabaseAdvisors.json

```diff
@@ -581,6 +581,11 @@
     "RecommendedAction": {
       "description": "Database, Server or Elastic Pool Recommended Action.",
       "type": "object",
+      "allOf": [
```

Comment:
what is the reason to change to proxy resource?

Reply:
Had a LintDiff error before that said there were conflicting definitions for RecommendedAction accross multiple swaggers.  Changed all to use ProxyResource for consistency.